bin/enrichEBEyeXMLDump.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python3
+
+"""
+This script adds to an Expression Atlas' EB-eye XML dump tissue and disease information retrieved 
+from condensed sdrf files. 
+Author: Robert Petryszak (info@datasome.co.uk)
+"""
+
+from xml.dom import minidom
+from re import sub, match, IGNORECASE
+import os
+import sys
+import time
+
+class EBEyeDumpEnrichmentError(Exception):
+    pass
+
+"""
+1. Create xml elemName
+2. Populate it with attributes in elemAttributeTuples, 
+3. Append to elemName a child text node elemText
+4. Append elemName as a child of parentElem
+"""
+def createAppendElement(doc, parentElem, elemName, elemText, elemAttributeTuples=[]):
```

Comment:
Python docstring should go inside functions when they are functions docs.

```suggestion
def createAppendElement(doc, parentElem, elemName, elemText, elemAttributeTuples=[]):
    """
    1. Create xml elemName
    2. Populate it with attributes in elemAttributeTuples, 
    3. Append to elemName a child text node elemText
    4. Append elemName as a child of parentElem
    """
```